class/defaults.yml

```diff
@@ -93,6 +93,9 @@ parameters:
       clusterAutoscaler: {}
       machineAutoscalers: {}
       priorityExpanderConfig: {}
+      metrics:
```

Comment:
Just an idle thought: maybe `custom_metrics` would be better, otherwise this might give the impression that the parameter controls the metrics configure of the cluster/machine-autoscaler rather than generating custom metrics.